core/src/main/antlr4/org/jdbi/v3/core/internal/lexer/SqlScriptLexer.g4

```diff
@@ -17,7 +17,8 @@ lexer grammar SqlScriptLexer;
 COMMENT
     : '--' ~('\n'|'\r')* |
       '//' ~('\n'|'\r')* |
-      '#'  ~('\n'|'\r')*
+      '#' ~('>'|'\r'|'\n') ~('\n'|'\r')* |
+      '#' ('\n'|'\r')
```

Comment:
It's cleaner to have a single expression for `#` comments, and make it conditional on whether the next character is `>`:

```antlr
COMMENT
    : '--' ~('\n'|'\r')* |
      '//' ~('\n'|'\r')* |
      {_input.LA(2) != '>'}? '#' ~('\n'|'\r')* // Exception for Postgres #> and #>> JSON operators
     { skip(); }
    ;
```

Reply:
Thanks, that's a much more elegant way of doing it. Changed.